src/app/util/clipboard.cpp

```diff
@@ -487,7 +504,9 @@ void Clipboard::paste(Context* ctx,
 
     case ClipboardFormat::Image: {
       // Get the image from the native clipboard.
-      if (!getImage(nullptr))
+      if (App::instance()->isGui() && !getImage(nullptr))
+        return;
```

Comment:
I think this change is not needed (?). Again a script could paste the native clipboard content if possible.